### PR TITLE
Mongo DB Setup

### PR DESCRIPTION
**Pull request by imraminradmand**
- Mongo setup in container
- Write function created to write both sessions and assets to the same collection

### CODE REVIEW

File: rest_golang/mongo/connectionhelper.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+//Used during creation of singleton client object in GetMongoClient().
+var clientInstanceError error
+
+//Used to execute client creation procedure only once.
+var mongoOnce sync.Once
+
+//I have used below constants just to hold required database config's.
+const (
+	// CONNECTIONSTRING = "mongodb://localhost:27017"
+	CONNECTIONSTRING = "mongodb://root:rootpassword@localhost:27017/?authSource=admin"
```
Comment (arsalanfardi) [inline rest_golang/mongo/connectionhelper.go:24]: Can this be moved to the .env file in the root directory of the project if possible? Let me know.

File: docker-compose.yml [hunk 2]
```diff
@@ -70,7 +70,18 @@ services:
       mindfuel_activity:
         ipv4_address: 10.10.20.1
 
+  mongodb_contaier:
```
Comment (arsalanfardi) [inline docker-compose.yml:73]: Let's remove the Elastic Search container from docker-compose.yml altogether (line 53-73).

File: mock_server/package-lock.json [hunk 3]
```diff
@@ -1,8 +1,96 @@
 {
   "name": "mock_server",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (arsalanfardi) [inline mock_server/package-lock.json:4]: Were the changes to package-lock.json necessary? If not, maybe we can revert it to the master version to clean up the PR a bit.

File: rest_golang/mongo/connectionhelper.go (hunk 1 above)
Comment (imraminradmand) [reply rest_golang/mongo/connectionhelper.go]: Moved to .env

**Review by arsalanfardi — APPROVED** [review]

**Pull request merged**